### ISSUE TITLE
Error: This resource requires GitHub organization to be set on the provider.

### ISSUE THREAD

**Issue by josjaf** [terraform-providers/terraform-provider-github#534]
terraform -v
Terraform v0.13.0
+ provider registry.terraform.io/-/github v2.9.2
+ provider registry.terraform.io/-/gitlab v2.11.0
+ provider registry.terraform.io/hashicorp/github v2.9.2
+ provider registry.terraform.io/terraform-providers/gitlab v2.11.0

I keep getting this error: Error: This resource requires GitHub organization to be set on the provider.

with this github provider

provider "github" {
  token = var.github_token

}

**Comment by gleichda** [comment]
@josjaf what resource are you using. Can you provide a bit more info. Like the [Issue Template](https://github.com/terraform-providers/terraform-provider-github/blob/master/.github/ISSUE_TEMPLATE.md) requires

**Comment by benjlevesque** [comment]
I'm getting this error too

### Terraform Version

``` 
terraform -v
Terraform v0.13.1
[...]
+ provider registry.terraform.io/hashicorp/github v2.9.2
``` 

### Affected resources
- github_repository_deploy_key

### Terraform Configuration Files: 

```hcl
provider "github" {
  token = var.github_token
}

resource "github_repository_deploy_key" "deploy_key" {
  title      = "deploy-key"
  repository = "myuser/myrepo"
  key        = tls_private_key.ssh.public_key_openssh
  read_only  = "true"
}
```

Error: This resource requires GitHub organization to be set on the provider.

**Comment by jcudit** [comment]
Fixed in [`v3.0.0`](https://github.com/terraform-providers/terraform-provider-github/blob/master/CHANGELOG.md#300-september-08-2020)
